storage/src/main/java/tech/pegasys/teku/storage/client/RecentChainData.java

```diff
@@ -196,7 +196,7 @@ public UInt64 getGenesisTimeMillis() {
   }
 
   public UInt64 computeTimeAtSlot(final UInt64 slot) {
-    return genesisTime.plus(slot.times(spec.getSecondsPerSlot(slot)));
+    return spec.atSlot(slot).miscHelpers().computeTimeAtSlot(genesisTime, slot);
```

Comment:
can't we just 
```suggestion
    return spec.computeTimeAtSlot(genesisTime, slot);
```

Reply:
yeh i guess we can now, mainly i dont want the math, so either works